src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html

```diff
@@ -358,7 +358,7 @@ <h3>Final disposition</h3>
             <span style="padding-left: 0.5rem;"
               *ngIf="(jjDisputedCount?.lesserOrGreaterAmount === jjDisputedCount?.ticketedFineAmount && inclSurcharge === 'yes')">{{
               jjDisputedCount?.lesserOrGreaterAmount | currency:'CAD':'symbol-narrow':'1.0-0' }}</span>
-            <span style="color: #33BBFF !important;"> + <i>({{ surcharge | currency }})</i></span>
+            <span style="color: #33BBFF !important;"> + <i>({{ surcharge | currency: 'CAD':'symbol-narrow':'1.0-0' }})</i></span>
```

Comment:
You can replace the default currency pipe to reduce lines of codes

Reply:
this is a good idea -- i have added a jira ticket TCVP-2354 for this.